source/common/router/config_impl.cc

```diff
@@ -287,6 +287,13 @@ RouteEntryImplBase::RouteEntryImplBase(const VirtualHostImpl& vhost,
                                     ? absl::optional<Http::LowerCaseString>(Http::LowerCaseString(
                                           route.route().host_rewrite_header()))
                                     : absl::nullopt),
+      host_rewrite_path_(
```

Comment:
nit: suggest using a "regex" in the naming so it's clear this is a regex

Reply:
IMHO that would make it less clear what the regex is being executed on. It seems like other options names suggest where the Host header rewrite is coming from, (e.g. `host_rewrite_literal`, `host_rewrite_header`) not how the result is constructed. 

With `host_rewrite_regex` you could imaging the regex being exectuted on some other header, or on the original Host name.

Having said that, happy to change if you think regex's better.

Reply:
Or, you mean `host_rewrite_path_regex`, not `host_rewrite_regex`?

Reply:
Haha yep, definitely agree that path should be included so it's clear. On my own first glance though the naming didn't line up to my understanding that this was a compiled regex matcher. But no worries either way (header file's got the type anyway)

Reply:
ok. changing to `host_rewrite_path_regex`